doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -160,10 +173,6 @@ The required number of connection points to a VM is described in `e.cap.004` abo
 | infra.hw.nac.cfg.002 | SmartNIC used to offload vSwitch functionality to hardware | Not required | Optional | |
```

Comment:
To keep the previous structure, extensions can be separated

Reply:
@karinesevilla Thanks. Added 

2.2.4.1 Cloud Infrastructure Hardware Profile-Extensions Requirements